servicecontrol/servicecontrol-instances/hardware.md

```diff
@@ -0,0 +1,61 @@
+---
+title: ServiceControl Hardware Considerations
+summary: Describes performance benchmarks performed for ServiceControl
```

Comment:
Is this an accurate summary? Maybe more something like

Hardware recommendations for running ServiceControl